### ISSUE TITLE
stopped working properly

### ISSUE THREAD

**Issue by artemax44** [p3t3rix-vsmods/VsProspectorInfo#35]
the same thing happens to me. The prospecting does not mark me. I'm using version 4.3.0 on 1.18.4 and it doesn't work correctly for me either.
It does not tell me the percentages or mark anything on the map.

with the one from version 1.18.5 the mod 4.3.0 doesn't work either, it still comes out the same

https://gyazo.com/70a5d4bf1a570c775ecdca825933c079

**Comment by Wooza** [comment]
There was a slight change in the prospecting output, which was adressed in #32. I suspect that the produced output depends on the version of the server. Do you know what version your server is running on? Additionally, what is the language of your game client?

**Comment by artemax44** [comment]
I don't know, I suppose the server version is the latest. I already tried to update the server but nothing. I don't think the language matters. I've had that mod since it came out and it never gave me any problems. only now from 1.18.4 and later 1.18.5, the previous versions worked very well for me

**Comment by artemax44** [comment]
ok I think the problem is not the mod, but the game itself. I have tried without loading the mod and the same thing happens to me. So I think it's a bug in the game update itself. I will continue to wait.

**Comment by p3t3rix** [comment]
Is this still an issue with 1.18.6-rc.1 ?

**Comment by Astrobirder** [comment]
I was having a similar issue as well.  Server is 1.18.3 (Mischief of Mice).  Client version was 4.2.0.  Suddenly new info quit showing up as overlay on the map.  Tried dropping the PP, switching picks, clearing the cache, nothing helped.  I then went and downloaded 4.3.0, staying with the 1.18.3 server.  Launched everything, still no go.  Deleted cache for both 4.2.0 and 4.3.0, relaunched and now it is working.
